packages/db/package.json

```diff
@@ -37,7 +37,7 @@
     "@t3-oss/env-nextjs": "^0.7.1",
     "dotenv-cli": "^7.3.0",
     "eslint": "^8.53.0",
-    "tsx": "^3.9.0",
+    "tsx": "4.7.2",
```

Comment:
Why was this changed?

Reply:
https://github.com/backstage/backstage/issues/21738

refer to this